src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -148,9 +148,34 @@
             "name": "starterTemplate",
             "type": "string",
             "description": "Starter template chosen during create document"
+        },
+        {
+            "name": "command",
+            "type": "string",
+            "description": "The id of a VS Code command"
+        },
+        {
+            "name": "debounceCount",
+            "type": "int",
+            "description": "Number of times the telemetry event was debounced before emission"
         }
     ],
     "metrics": [
+        {
+            "name": "vscode_executeCommand",
+            "description": "Emitted whenever a registered Toolkit command is executed",
+            "passive": true,
+            "metadata": [
+                { "type": "command" },
+                { "type": "debounceCount" },
```

Comment:
the debounce seems like an implementation detail. Is the field really just `count`? 

counterargument is that we have successCount and failureCount, which refer to sub-operations of a single "action": https://github.com/aws/aws-toolkit-common/blob/448a2d4c555abad138d459e3831e29d3756ae0ac/telemetry/definitions/commonDefinitions.json#L363-L372

so naming this field `debounceCount` makes it clear that the metric was collapsed. Still the name seems a bit low-level. Probably `count` or `invocationCount` can be established with a clear meaning.

Reply:
Yeah you're right that this isn't a whole lot different from a normal `count`. I guess I didn't want anyone to think that a count of X meant that X metrics _with the same data_ would've been emitted.